src/geocat/viz/util/util.py

```diff
@@ -725,17 +870,28 @@ def plotELabels(da,
     return extremaLabels
 
 
-def set_vector_density(data, minDistance=0):
+def set_vector_density(data: xarray.DataArray, 
+                       minDistance: int = 0) -> xarray.DataArray:
     """Utility function to change density of vector plots.
 
-    Args:
-        data (:class:`xarray.core.dataarray.DataArray`):
-            Data array that contains the vector plot latitude/longitude data.
-        minDistance (:class:`int`):
-            Value in degrees that determines the distance between the vectors.
-    Returns:
-        ds (:class:`xarray.core.dataarray.DataArray`):
-            Sliced version of the input data array.
+    Parameters
+    ----------
+    data: :class:`xarray.DataArray`
+        Data array that contains the vector plot latitude/longitude data.
+    
+    minDistance: :class:`int`
+        Value in degrees that determines the distance between the vectors.
+    
+    Returns
+    -------
+    ds: :class:`xarray.DataArray`
+        Sliced version of the input data array.
+        
+    Examples
+    --------
+    All usage examples are within the GeoCAT-Examples Gallery. To see more usage cases, search the function on the `website <https://geocat-examples.readthedocs.io/en/latest/index.html>`_.
+    
+    - `NCL_vector_3.py <https://geocat-examples.readthedocs.io/en/latest/gallery/Vectors/NCL_vector_3.html?highlight=set_vector_density>`_
     """
```

Comment:
Another "not in the PR, just left over thing to be addressed":
Line 917 initializes `ds = data` and then ds is unused. So it would be fine to remove this line from the code.